historian/backend/src/main/java/com/hurence/logisland/historian/config/SecurityConfigTest.java

```diff
@@ -0,0 +1,91 @@
+//package com.hurence.logisland.historian.config;
```

Comment:
Is it really needed since it's all commented out ?

Reply:
You are rigth, I fixed this in another branch.